environment-api/src/test/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatusTest.java

```diff
@@ -34,6 +34,11 @@
                 { "UPDATE_FAILED",                          EnvironmentStatus.UPDATE_FAILED,                        false },
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS, false },
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS, false },
+                { "UPGRADE_CCM_VALIDATION_FAILED",          EnvironmentStatus.UPGRADE_CCM_VALIDATION_FAILED,        false },
+                { "UPGRADE_CCM_ON_FREEIPA_FAILED",          EnvironmentStatus.UPGRADE_CCM_ON_FREEIPA_FAILED,        false },
+                { "UPGRADE_CCM_ON_DATALAKE_FAILED",         EnvironmentStatus.UPGRADE_CCM_ON_DATALAKE_FAILED,       false },
+                { "UPGRADE_CCM_ON_DATAHUB_FAILED",          EnvironmentStatus.UPGRADE_CCM_ON_DATAHUB_FAILED,        false },
+                { "UPGRADE_CCM_FAILED",                     EnvironmentStatus.UPGRADE_CCM_FAILED,                   false },
         };
```

Comment:
What about the new `*_IN_PROGRESS` and `UPGRADE_CCM_ROLLING_BACK` states?

Reply:
added